src/arrayMethodJoin.js

```diff
@@ -5,7 +5,22 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```

You can set a default value for separator